run.go

```diff
@@ -26,6 +26,7 @@ type Run struct {
 	Tools          []Tool             `json:"tools"`
 	FileIDS        []string           `json:"file_ids"`
 	Metadata       map[string]any     `json:"metadata"`
+	Usage          *RunUsage          `json:"usage,omitempty"`
```

Comment:
I'm not sure we need a separate `RunUsage` type here.

Also, let's not use a pointer here, see other `Usage` examples in the repo: https://github.com/search?q=repo%3Asashabaranov%2Fgo-openai%20Usage&type=code

Reply:
@shadowpigy and a minor change: please remove the pointer

Reply:
@shadowpigy hey, waiting for small change from you and we're good to go!